chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -149,32 +149,32 @@
 
       package "grub2-#{grub2arch}-efi"
 
-      # grub.cfg has to be in boot/grub/ subdirectory
-      directory "#{uefi_dir}/default/boot/grub" do
+      # grub.cfg has to be in boot/grub2/ subdirectory
+      directory "#{uefi_dir}/boot/grub2" do
         recursive true
         mode 0755
         owner "root"
         group "root"
         action :create
       end
 
-      template "#{uefi_dir}/default/boot/grub/grub.cfg" do
+      template "#{uefi_dir}/boot/grub2/grub.cfg" do
```

Comment:
This is not extra subdir, we would like to replace old path `default/boot/grub` with new one `boot/grub2`